deploy/15_avatar/01_deploy_avatar_v1.ts

```diff
@@ -0,0 +1,40 @@
+// TODO: Validate if we want a L1 avatar contract ?
+
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {skipUnlessTestnet} from '../../utils/network';
+
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {deployer, upgradeAdmin, sandAdmin} = await getNamedAccounts();
+
+  const TRUSTED_FORWARDER = await deployments.get('TRUSTED_FORWARDER');
+  const adminRole = sandAdmin;
+  await deployments.deploy('Avatar', {
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+    proxy: {
+      owner: upgradeAdmin,
+      proxyContract: 'OptimizedTransparentProxy',
```

Comment:
'OpenZeppelinTransparentProxy' is used in other places - what is our preference?